src/components/AllFeedScreen.tsx

```diff
@@ -24,29 +25,35 @@ export interface StateProps {
 
 type Props = StateProps & DispatchProps;
 
-export const AllFeedScreen = (props: Props) => {
-    const modelHelper = new ReactNativeModelHelper(props.gatewayAddress);
-    return (
-        <RefreshableFeed modelHelper={modelHelper} {...props}>
-            {{
-                navigationHeader: <NavigationHeader
-                                title='All feeds'
-                                rightButtonText1={
-                                    <Icon
-                                        name={'view-grid'}
-                                        size={20}
-                                        color={Colors.DARK_GRAY}
-                                    />
-                                }
-                                onPressRightButton1={() => props.navigation.navigate('FeedListViewerContainer')}
-                            />,
-                listHeader: <FeedHeader
-                                navigation={props.navigation}
-                                onSaveDraft={props.onSaveDraft}
-                                profileImage={props.profileImage}
-                                gatewayAddress={props.gatewayAddress}
-                            />,
-            }}
-        </RefreshableFeed>
-    );
-};
+export class AllFeedScreen extends React.Component<Props> {
+    public render() {
+        const modelHelper = new ReactNativeModelHelper(this.props.gatewayAddress);
+        return (
+            <RefreshableFeed modelHelper={modelHelper} {...this.props}>
+                {{
+                    navigationHeader: <NavigationHeader
+                                    title='All feeds'
+                                    rightButtonText1={
+                                        <Icon
+                                            name={'view-grid'}
+                                            size={20}
+                                            color={Colors.DARK_GRAY}
+                                        />
+                                    }
+                                    onPressRightButton1={() => this.props.navigation.navigate('FeedListViewerContainer')}
+                                />,
+                    listHeader: <FeedHeader
+                                    navigation={this.props.navigation}
+                                    onSaveDraft={this.props.onSaveDraft}
+                                    profileImage={this.props.profileImage}
+                                    gatewayAddress={this.props.gatewayAddress}
+                                />,
+                }}
+            </RefreshableFeed>
+        );
+    }
+
+    public componentDidMount() {
```

Comment:
Why did you choose to put it here? I understand that right now this is the first screen that we show if you are not onboarding, however this may change if we change navigation.

Did you try putting it into the App itself?